logic/transaction.js

```diff
@@ -36,21 +36,19 @@ const __private = {};
  * @requires extend
  * @requires lodash
  * @requires helpers/bignum
- * @requires helpers/constants
- * @requires helpers/exceptions
  * @requires helpers/slots
  * @param {Database} db
  * @param {Object} ed
  * @param {ZSchema} schema
- * @param {Object} genesisblock
+ * @param {Object} genesisBlock
```

Comment:
If we do this change, we should do it across the whole code. There are a few places in the test suite where we still call it `genesisblock`.

Reply:
I tried to fix it across the code base. Will see where it is missed.